src/main/resources/htmlpublisher/HtmlPublisher/header.html

```diff
@@ -81,7 +81,7 @@
     tab.setAttribute("class", "selected");
     selectedTab = tabId;
     iframe = document.getElementById("myframe");
-    iframe.src = encodeURIComponent(tab.getAttribute("value"));
+    iframe.src = encodeURIComponent(tab.getAttribute("value")).replace('%2F', '/');
```

Comment:
We just ran into an issue with paths that contain multiple slashes, such as `foo/bar/main.html`. In this case, only the first occurrence of `%2F` is replaced, i.e. `iframe.src` ends up as `foo/bar%2Fmain.html`. As a result, all relative links on that page are now causing a 404.

Using a regular expression as the search pattern, as in `replace(/%2F/g, '/')`, would replace all occurrences of the search pattern.

Reply:
sounds good idea. Can you make a Pull request?

Reply:
Sure thing: https://github.com/jenkinsci/htmlpublisher-plugin/pull/146

However, not having a Java development environment at hand, I'm not able to verify that my changes to the corresponding unit test will actually work as intended.